assets/app/views/_templateopt.html

```diff
@@ -0,0 +1,21 @@
+<div class="template-options gutter-bottom" ng-show="hasParameters">
+  <div class="flow">
+    <div class="flow-block">
+      <h2>Template options</h2>
+    </div>
+    <div class="flow-block right">
```

Comment:
I guess the only reason would be to make the page shorter so you can get to the buttons below, but yeah I think it'd be fine without it. The wording for it is misleading as well. 'Collapse' would be a better label.